cmd/ocsp-updater/main.go

```diff
@@ -395,25 +406,51 @@ func (updater *OCSPUpdater) missingReceiptsTick(batchSize int) {
 			continue
 		}
 	}
-
+	return nil
 }
 
 type looper struct {
-	clk       clock.Clock
-	stats     statsd.Statter
-	batchSize int
-	tickDur   time.Duration
-	tickFunc  func(int)
-	name      string
+	clk                  clock.Clock
+	stats                statsd.Statter
+	batchSize            int
+	tickDur              time.Duration
+	tickFunc             func(int) error
+	name                 string
+	recentFailures       int64
```

Comment:
This is exported by the RPC layer, but is not used for stats here. This var is used to pass into `core.RetryBackoff` as the `retries` param, so with each error the backoff of the loop is increased. The goroutine spawned at L430 resets the count to `0` every hour so that the backoff will reset between HSM failures so we don't get the desired increasing backoff just once.